src/Services/OrganizationService.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace OpenEMR\Services;
+
+use OpenEMR\FHIR\R4\FHIRDomainResource\FHIROrganization;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRAddress;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRCodeableConcept;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRCoding;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRId;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRMeta;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRReference;
+use OpenEMR\FHIR\R4\FHIRResource\FHIRBundle;
+use OpenEMR\FHIR\R4\FHIRResource\FHIRBundle\FHIRBundleLink;
+use OpenEMR\FHIR\R4\PHPFHIRResponseParser;
+
+class OrganizationService
```

Comment:
Naming convention
FHIROrganizationService

May not be issue now but if we add similar to standard api then issue